src/synth/fluid_synth.c

```diff
@@ -3367,6 +3428,53 @@ fluid_synth_render_blocks(fluid_synth_t* synth, int blockcount)
   return blockcount;
 }
 
+/*
+ * Handler for synth.reverb.* settings.
```

Comment:
Maybe also mention synth.chorus.* settings and explain the _num (floating point) and _int (int) distinction?

Reply:
ok